### ISSUE TITLE
Reply to a message - feature request

### ISSUE THREAD

**Issue by chiumichael** [keybase/keybase-issues#3057]
Would be very helpful if one could reply to a message like in whatsapp.

**Comment by fiatjaf** [comment]
On Keybase Chat? That's exactly the only thing you can do with a message you receive already: reply it.

**Comment by chiumichael** [comment]
Yes, I mean like in slack and whatsapp, where you can "reply" to an earlier message, and have that message quoted. Slack and whatsapp both have this feature and it's pretty useful imo when chats get wieldly.

**Comment by fiatjaf** [comment]
Oh, I understand. That is indeed a good feature to have.

I would also like audio messages like WhatsApp (and none other messaging app that I've tried to use) has. That would be huge (but very complicated to implement).

**Comment by chiumichael** [comment]
tx, will defo rename it!

Agreed with the audio messages but for now at least, I'll be happy with a reply with quote feature =)

**Comment by sci4me** [comment]
Discord lets you type `>>>` to quote things -- why not Keybase? Come on ya'll, it's 2021.
